python/src/aac/plugins/first_party/gen_plugin/templates/third_party/__init__.py.jinja2

```diff
@@ -54,7 +55,7 @@ def _get_plugin_commands():
         {% for command in commands %}
         AacCommand(
             "{{command.name}}",
-            "{{command.description}}",
+            "{{command.helpText}}",
```

Comment:
Meh, we could buy it's not intended to be a 1:1 making with the yaml

Reply:
Fair, but because of how the documentation got worded with the 2.0 release I thought they were meant to be. Might be worth adding it to the list for discussion of baselining efforts for 23.3.